src/wx/viewsupt.cpp

```diff
@@ -250,7 +249,7 @@ void DisList::SetSel()
             start += i;
             end += i + 1;
 #endif
-            tc.SetSelection(start, start + strings[i].size() + 1);
+            tc.SetSelection(start, end);
```

Comment:
I think the intent here was to have this argument be `end`, so I changed it to that.

Does that seem right?